README.md

```
@@ -1,469 +1,312 @@
-    {
-      event_type: 'alerts'
-    }
-  ]
-}));
-
-ws.onmessage = (event) => {
-  const message = JSON.parse(event.data);
-  console.log('Live update:', message);
-};
+zfs://pools/squirts
+zfs://snapshots/squirts?limit=100
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)